src/jobtypes.py

```diff
@@ -3,8 +3,8 @@
 import logging
 from tools.update_runnablejobs import update_runnableapi
 LOG = logging.getLogger(__name__)
-# JSONPATH = os.getcwd() + '/runnablejobs.json'
-JSONPATH = '/home/ubuntu/ouija/data/runnablejobs.json'
+JSONPATH = os.getcwd() + '/runnablejobs.json'
+# JSONPATH = '/home/ubuntu/ouija/data/runnablejobs.json'
```

Comment:
I like all the changes except this one- this will not work for alert manager- I would prefer to leave this out for now until we can configure the jsonpath better, ideally in a data/ subdir for all input data like runnable_jobs.json, preseed.json, etc...